helm/default-apps-aws/templates/apps.yaml

```diff
@@ -6,6 +6,10 @@ kind: App
 metadata:
   annotations:
     chart-operator.giantswarm.io/force-helm-upgrade: "{{ .forceUpgrade }}"
+    {{- if .dependsOn }}
+    # App is deployed in the org- namespace so the secret name is prefixed by the cluster-id
```

Comment:
what secret name you mean here? I don't understand the comment

Reply:
@QuentinBisson

Reply:
Comment updated